src/front/ticketer/lib/pages/admin/landing/organizer_listing.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+import 'package:ticketer/pages/common/organizer_card.dart';
+
+class OrganizerListing extends StatefulWidget {
+  const OrganizerListing({Key? key}) : super(key: key);
+
+  @override
+  State<OrganizerListing> createState() => _OrganizerListingPageState();
+
+}
+
+class _OrganizerListingPageState  extends State<OrganizerListing>  {
+
+  Widget _getContent() {
+    return SingleChildScrollView(
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.center,
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          _organizerListItem(),
+          _organizerListItem(),
+          _organizerListItem(),
+        ],
+      ),
+    );
+  }
+
+  Widget _organizerListItem() {
+    return Row(
+      crossAxisAlignment: CrossAxisAlignment.center,
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
```

Comment:
I think we need to establish consistent order of buttons (positive - negative). I think it doesn't matter which convention we use as long as we use colors, but it will be good to have it consistent (it differs here and in sector adding on organizer landing page).

![image](https://user-images.githubusercontent.com/85129282/229342478-94dbecc8-5a4f-4f2d-ab1b-b10150974f0e.png)

Reply:
Agreed with the team to go with Apple guidelines.